spec/controllers/api/users_controller_spec.rb

```diff
@@ -22,4 +27,23 @@
       expect(response).to have_http_status(401)
     end
   end
+
+  describe 'GET show' do
+    it 'should return unauthorized if not logged in' do
+      get :show, params: {
+        id: @user1.id
+      }
+
+      expect(response).to have_http_status(:unauthorized)
+    end
+
+    it 'should return ok if golfer logged in' do
+      sign_in(@user1)
+      get :show, params: {
+        id: @user1.id
+      }
+
+      expect(response).to have_http_status(:ok)
+    end
```

Comment:
Check also if the response contains the requested user. For example, you can verify if the id is the same.

Reply:
Added new expect cases based on response body.